README.md

```diff
@@ -67,22 +67,27 @@ If you don't currently have Rails installed (or are on Windows), Cloud9 makes th
 
 ### Docker
 
+<<<<<<< HEAD
```

Comment:
Can you take a look at this merge conflict? Colin made a couple small changes (just that spelling error and the added link) so go ahead and keep those. Thanks!